api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
```

Comment:
How about `./project.rb` and `libproject` for the folder? That lets you easily tab-complete `./project.rb` without hiding the helper scripts.